sdk/key_vault/src/secret.rs

```diff
@@ -72,6 +72,9 @@ pub(crate) struct KeyVaultGetSecretResponse {
 #[derive(Deserialize, Debug)]
 pub(crate) struct KeyVaultGetSecretResponseAttributes {
     enabled: bool,
+    #[serde(default)]
+    #[serde(with = "ts_seconds_option")]
+    exp: Option<DateTime<Utc>>,
```

Comment:
Following our other SDKs (I'll be updating the rest of the fields here eventually), this should be `not_before` similar to our [Python SDK](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-keyvault-secrets/latest/azure.keyvault.secrets.html#azure.keyvault.secrets.SecretClient.set_secret). Other SDKs use NotBefore or notBefore depending on casing.

Reply:
Same with values below.

Reply:
According to that python SDK link, should the expiry field not be `expires_on` instead?

Reply:
Yes, sorry. I was thinking of the wrong member (nbf).